static/releases.json

```diff
@@ -1 +1 @@
{"version":"3.1.3"},{"version":"4.0.4"},{"version":"5.0.0-beta.2"},{"version":"4.0.5"},{"version":"5.0.0-beta.3"},{"version":"3.1.4"},{"version":"4.0.6"},{"version":"5.0.0-beta.4"},{"version":"3.1.5"},{"version":"4.0.7"},{"version":"5.0.0-beta.5"},{"version":"2.0.18"},{"version":"3.1.6"},{"version":"3.0.16"},{"version":"4.0.8"},{"version":"4.1.0"},{"version":"5.0.0-beta.6"},{"version":"4.1.1"},{"version":"3.1.7"},{"version":"5.0.0-beta.7"},{"version":"3.1.8"},{"version":"4.1.2"},{"version":"4.1.3"},{"version":"5.0.0-beta.8"},{"version":"4.1.4"},{"version":"5.0.0-beta.9"},{"version":"5.0.0"},{"version":"4.1.5"},{"version":"6.0.0-beta.1"},{"version":"3.1.9"},{"version":"4.2.0"},{"version":"5.0.1"},{"version":"6.0.0-beta.2"},{"version":"4.2.1"},{"version":"6.0.0-beta.3"},{"version":"4.2.2"},{"version":"6.0.0-beta.4"},{"version":"5.0.2"},{"version":"3.1.10"},{"version":"6.0.0-beta.5"},{"version":"4.2.3"},{"version":"6.0.0-beta.6"},{"version":"4.2.4"},{"version":"3.1.11"},{"version":"5.0.3"},{"version":"6.0.0-beta.7"},{"version":"6.0.0-beta.8"},{"version":"5.0.4"},{"version":"6.0.0-beta.9"},{"version":"5.0.5"},{"version":"4.2.5"},{"version":"6.0.0-beta.10"},{"version":"6.0.0-beta.11"},{"version":"5.0.6"},{"version":"4.2.6"},{"version":"6.0.0-beta.12"},{"version":"6.0.0-beta.13"},{"version":"3.1.12"},{"version":"4.2.7"},{"version":"5.0.7"},{"version":"6.0.0-beta.14"},{"version":"4.2.8"},{"version":"6.0.0-beta.15"},{"version":"5.0.8"},{"version":"6.0.0"},{"version":"3.1.13"},{"version":"7.0.0-beta.1"},{"version":"4.2.9"},{"version":"5.0.9"},{"version":"6.0.1"},{"version":"7.0.0-beta.2"},{"version":"6.0.2"},{"version":"7.0.0-beta.3"},{"version":"5.0.10"},{"version":"6.0.3"},{"version":"6.0.4"},{"version":"6.0.5"},{"version":"4.2.10"},{"version":"6.0.6"},{"version":"6.0.7"},{"version":"7.0.0-beta.4"},{"version":"6.0.8"},{"version":"6.0.9"},{"version":"6.0.10"},{"version":"4.2.11"},{"version":"7.0.0-beta.5"},{"version":"5.0.11"},{"version":"6.0.11"},{"version":"6.0.12"},{"version":"7.0.0-beta.6"},{"version":"7.0.0-beta.7"},{"version":"4.2.12"},{"version":"7.0.0"},{"version":"6.1.0"},{"version":"6.1.1"},{"version":"8.0.0-beta.1"},{"version":"6.1.2"},{"version":"8.0.0-beta.2"},{"version":"6.1.3"},{"version":"7.0.1"},{"version":"5.0.12"},{"version":"6.1.4"},{"version":"7.1.0"},{"version":"7.1.1"},{"version":"7.1.2"},{"version":"8.0.0-beta.3"},{"version":"6.1.5"},{"version":"7.1.3"},{"version":"8.0.0-beta.4"},{"version":"7.1.4"},{"version":"6.1.6"},{"version":"7.1.5"},{"version":"6.1.7"},{"version":"5.0.13"},{"version":"7.1.6"},{"version":"8.0.0-beta.5"},{"version":"7.1.7"},{"version":"7.1.8"},{"version":"7.1.9"},{"version":"8.0.0-beta.6"},{"version":"8.0.0-beta.7"},{"version":"7.1.10"},{"version":"8.0.0-beta.8"},{"version":"8.0.0-beta.9"},{"version":"7.1.11"},{"version":"8.0.0"},{"version":"9.0.0-beta.1"},{"version":"7.1.12"},{"version":"8.0.1"},{"version":"9.0.0-beta.2"},{"version":"7.1.13"},{"version":"6.1.8"},{"version":"9.0.0-beta.3"},{"version":"8.0.2"}]
```

Comment:
Remind me why we don't just add a dependency on [`electron-releases`](https://github.com/electron/releases) ?

Reply:
We don't shake our dependencies and the full `electron-releases` weighs a whopping 48 mb!

Reply:
That would do it.

Reply:
I wrote a GitHub Action to automate this for us: https://github.com/electron/fiddle/pull/331